src/services/solana.ts

```diff
@@ -97,14 +97,16 @@ export function getConnection(cluster = process.env.SOLANA_CLUSTER) {
 export function makeMetaplex() {
   const connection = getConnection();
   const metaplex = Metaplex.make(connection)
+    .use(keypairIdentity(getKeyPair("AUTHORITY")));
```

Comment:
are we even using this function anywhere?

Reply:
@wasif1024 have you had a chance to take a look at where this function is being referenced?

Reply:
yes this is being used in mint workflow.